### PR TITLE
HOTFIX: Negative Static Margin

### PR DESCRIPTION
**Pull request by giovaniceotto**
## Pull request type

- [x] Code changes (bugfix, features)

## Checklist

- [x] Tests for the changes have been added (if needed)
- [x] Lint (`black rocketpy/ tests/`) has passed locally 
- [x] All tests (`pytest --runslow`) have passed locally

## Current behavior

When using the `nose_to_tail` coordinate system for the `Rocket` class, the sign of the static margin was inverted since a sign correction was applied twice, as noticed by @MateusStano.

## New behavior

Things just work now. One of the sign corrections was removed. Ironically, we had tests for this, but they were not working properly. This has also been fixed.

Version has been bumped up to v1.1.1 so that a hotfix can be released.

## Breaking change

- [x] No

## Additional information

This bug was reported by Bob Brown [D&W] (MΦNK3Y#2638) through our Discord server.

### CODE REVIEW

**Review by Gui-FernandesBR — APPROVED** [review]
This Looks Good To Me

**Review by phmbressan — APPROVED** [review]

File: tests/test_rocket.py [hunk 1]
```diff
@@ -98,13 +98,9 @@ def test_coordinate_system_orientation(
...
     rocket_nose_to_tail.aerodynamic_surfaces.add(calisto_trapezoidal_fins, 1.168)
 
-    static_margin_nose_to_tail = rocket_nose_to_tail.static_margin(0)
+    static_margin_nose_to_tail = rocket_nose_to_tail.static_margin
 
-    assert (
-        rocket_tail_to_nose.all_info() == None
-        or rocket_nose_to_tail.all_info() == None
-        or not abs(static_margin_tail_to_nose - static_margin_nose_to_tail) < 0.0001
-    )
```
[the remark this answers is not in the document]
Comment (Gui-FernandesBR) [reply tests/test_rocket.py:107]: you're right. The decorator and the mock should be dropped.

[the remark this answers is not in the document]
Comment (giovaniceotto) [reply tests/test_rocket.py:107]: Good catch @phmbressan. Just fixed it in https://github.com/RocketPy-Team/RocketPy/pull/476/commits/f3988751189164ae12ae58f666a0f3988394c16a.

**Comment by Gui-FernandesBR** [comment]
@giovaniceotto I think u can merge this one already.

Could u create release notes pls?

**Comment by giovaniceotto** [comment]
>@giovaniceotto I think u can merge this one already.

I will let the boss @MateusStano do it in case there is anything else relevant that I am not aware of.

> Could u create release notes pls?

I wanted to create it in the file, but it isn't in master yet. So I'll just do it here to help the release:

### Fixed

- Opposite static margin bug, which affected all rockets defined using a `nose_to_tail` coordinate system, causing positive static margins to become negative and negative ones to become positive. Note: this bug did not lead to any issues in terms of the trajectory simulation itself.

**Pull request merged**